libs/perun/utils/src/lib/export-table-dialog/export-table-dialog.component.html

```diff
@@ -0,0 +1,49 @@
+<h1 mat-dialog-title>
+  {{'SHARED_LIB.PERUN.COMPONENTS.TABLE_OPTIONS.EXPORT_TITLE'| translate}}
+</h1>
+<div mat-dialog-content class="dialog-container user-theme">
+  <mat-form-field>
+    <mat-label>{{'SHARED_LIB.PERUN.COMPONENTS.TABLE_OPTIONS.SELECT_FORMAT'| translate}}</mat-label>
+    <mat-select [formControl]="selectedFormat">
+      <mat-option *ngFor="let format of formats" [value]="format.value">
+        {{format.viewValue}}
+      </mat-option>
+    </mat-select>
+  </mat-form-field>
+
+  <p>{{'SHARED_LIB.PERUN.COMPONENTS.TABLE_OPTIONS.EXPORT_OPTIONS'| translate}}</p>
+
+  <mat-radio-group class="mb-2" required [formControl]="selectedExportType">
+    <mat-radio-button value="current" class="flex-column d-flex" color="primary">
+      {{'SHARED_LIB.PERUN.COMPONENTS.TABLE_OPTIONS.DISPLAYED_DATA'| translate}}
+    </mat-radio-button>
+    <mat-radio-button
+      *ngIf="inputData.allowExportAll"
+      value="all"
+      class="flex-column d-flex"
+      color="primary">
+      {{'SHARED_LIB.PERUN.COMPONENTS.TABLE_OPTIONS.ALL_DATA'| translate}}
+    </mat-radio-button>
+  </mat-radio-group>
```

Comment:
Nitpick but this would better represent the layout.
```suggestion
  <mat-radio-group class="flex-column d-flex mb-2" required [formControl]="selectedExportType">
    <mat-radio-button value="current" color="primary">
      {{'SHARED_LIB.PERUN.COMPONENTS.TABLE_OPTIONS.DISPLAYED_DATA'| translate}}
    </mat-radio-button>
    <mat-radio-button
      *ngIf="inputData.allowExportAll"
      value="all"
      color="primary">
      {{'SHARED_LIB.PERUN.COMPONENTS.TABLE_OPTIONS.ALL_DATA'| translate}}
    </mat-radio-button>
  </mat-radio-group>
```